Dockerfile

```diff
@@ -1,8 +1,14 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+MAINTAINER hello@codeclimate.com
 
 RUN adduser -u 9000 -D app
-USER app
 
-CMD ["/usr/src/app/codeclimate-gofmt"]
+WORKDIR /code
+
+COPY build/ /usr/src/app
+COPY build/codeclimate-gofmt /usr/src/app/build/codeclimate-gofmt
```

Comment:
I feel like we shouldn't actually need any contents of `build` other than the bin? That was the intent of the older `ADD` ccommand. Could we just have this last COPY without the one before it?